### ISSUE TITLE
Question about face keypoints name

### ISSUE THREAD

**Issue by buaacyw** [mks0601/Hand4Whole_RELEASE#47]
Thanks for your great work!
I have a question about face key points names.
![image](https://user-images.githubusercontent.com/52091468/201655666-73c61e61-52a8-42b5-8e10-8b66c395ed8e.png)

In human_models.py, it's mentioned that face key points names can be found in FLAME class.
But I didn't find it.
I have searched about FLAME face landmark orders but I can't find it.
So could you please help me to find the FLAME face landmark orders that your model uses? (Like the below image) 
![image](https://user-images.githubusercontent.com/52091468/201655807-24e994b6-f4b0-4ba9-926c-e311482350cd.png)

 Thanks！

**Comment by mks0601** [comment]
Those names are made by me, and actually FLAME does not provide any images like that.. So I just manually checked all landmarks..
One thing is that you can check here: https://github.com/mks0601/Hand4Whole_RELEASE/blob/Pose2Pose/data/MSCOCO/MSCOCO.py
It provides compatible names with those of FLAME, so you can check it. You can check MSCOCO facial keypoints at https://github.com/jin-s13/COCO-WholeBody

**Comment by buaacyw** [comment]
Thanks for your quick reply! It really helps.
So just to make sure:
![image](https://user-images.githubusercontent.com/52091468/201810269-9b83911d-7dd4-4a17-97a1-bf34a09c4f2f.png)
`joint_to_flame[flame_landmark_index]=coco_index`
in which flame_landmark_index ranges from 0 to 72 (this is precisely the 73 total face landmark numbers mentioned in [human_models.py](https://github.com/mks0601/Hand4Whole_RELEASE/blob/2b809045d96aa9adb1d119372c9a10b63b575d7e/common/utils/human_models.py#L54)). And except -1, coco_index range from 0 to 67. Add 24 to each coco_index, we can get the same index in the below image.
![image](https://user-images.githubusercontent.com/52091468/201810283-75ef0ee4-3494-4d7e-8273-2c4d908ea854.png)

Is this right? Thanks!

**Comment by mks0601** [comment]
Maybe, but you may want to check it my yourself. It's been a while, so I can't remember clearly

**Comment by buaacyw** [comment]
Thanks!

**Comment by felixshing** [comment]
Hi guys, I would like ask a relevant question. Given that we can use a lot of keypoints on human's face, why we still resort to SMPL-X, which contains only 10 keypoints for facial expressions at the end?

**Comment by mks0601** [comment]
Could you elaborate? I have no idea what you mean by 10 keypoints. there are hundreads of mesh vertices in SMPL-X.

**Comment by felixshing** [comment]
Oh I mean SMPL-X has 10 parameters for controlling facial parameters. I thought they mean it has 10 keypoints on the face. Am I understanding it correctly?

What is the purpose for registering SMPL-X model here? Is it for evaluation or for generating human model? 

I was wondering, since we can have a lot of keypoints, why we cannot just use them to generate pose but still resort to SMPL-X, which may lose some details (I am not sure whether such claim is true.) If we want our generated human model is photo-realistic, should we choose not to rely on SMPL-L or we can still use it?

**Comment by mks0601** [comment]
No. 10-dimensional vector represents a low-dimensional embedding of the facial expression like coeffiecient of the PCA space.
keypoint is a subset of mesh vertices. each vertex of a mesh can be seen as a keypoint.

**Comment by felixshing** [comment]
Oh I see. So will adding some keypoints on face, such as using google' media pipe, help improve facial expressions when we use SMPL-X? And can we generate photo-realistic model with SMPL-X?
